CONTRIBUTING.md

```diff
@@ -0,0 +1,64 @@
+# Contributing to Drone Controller
+
+First off all, thank you very much for taking interest in contributing to our project. We value your time and effort.
+
+The following guideline shall help to development to create a save and efficient environment to develop open source software.
+
+## Questions
+
+We try to document as much as possible but sometimes things are missed. 
```

Comment:
```suggestion
We try to document as much as possible but sometimes things are missing.  
```

Reply:
After consultation with I think it missed.